zc_common/remote_resource/models.py

```diff
@@ -51,3 +52,95 @@ def deconstruct(self):
         del kwargs['max_length']
 
         return name, path, args, kwargs
+
+
+class GenericRemoteForeignKey(object):
```

Comment:
This has a lot of logic that are not easy to verify, how have we tested this? are we planning to write tests for it? How confident are we that this is stable and reliable?